src/unix/process.c

```diff
@@ -379,6 +382,20 @@ static void uv__process_child_init(const uv_process_options_t* options,
       uv__close(close_fd);
   }
 
+  if (options->flags & UV_PROCESS_PTY) {
+    // Put ourself into a new session and process group, making us session
+    // and process group leader.
+    if (setsid() < 0) {
+        uv__write_errno(error_fd);
+    }
```

Comment:
Style issues: C89-style comments and no braces around single line `if` statements.
```suggestion
    /* Put ourself into a new session and process group, making us session
      * and process group leader.
      */
    if (setsid() < 0)
        uv__write_errno(error_fd);
```